### PR TITLE
Bump webjars-locator-core to 0.46

### LINKED ISSUE #10313: Webjars-locator only works in quarkus:dev mode
**Describe the bug**
New extension quarkus-webjars-locator in 1.6.0.CR1 should be fairly straightforward to use. It works as expected in 
quarkus:dev mode, but in a jar/docker container it doesn't seem to work (if I'm not missing something). 

Reproducer can be found here: 
https://github.com/mejlholm/webjars-locator-reproducer

**Expected behavior**
Web application on localhost:9080/index.html finds all webjar included dependencies without versions. 

**Actual behavior**
404 - not found. 

**To Reproduce**
Build and run jar/container.

**Configuration**
Nothing special.

**Screenshots**
In dev mode
billede
In jar/docker container
billede

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: 
Microsoft Windows [Version 10.0.19041.329]

 - Output of `java -version`: 
java version "1.8.0_251"
Java(TM) SE Runtime Environment (build 1.8.0_251-b08)
Java HotSpot(TM) 64-Bit Server VM (build 25.251-b08, mixed mode)

 - Quarkus version or git rev: 
1.6.0.CR1
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): 
Apache Maven 3.6.2 (40f52333136460af0dc0d7232c0dc0bcf0d9e117; 2019-08-27T17:06:16+02:00)
Maven home: C:\dev\tools\apache-maven-3.6.2\bin\..
Java version: 11.0.2, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-11.0.2
Default locale: en_US, platform encoding: Cp1252

OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

**Additional context**
(Add any other context about the problem here.)

### PR DESCRIPTION
**Pull request by gastaldi**
Fixes #10313

### CODE REVIEW

**Comment by gastaldi** [comment]
This PR also changes the webjar-locator extension to depend only on the `webjars-location-core` which contains only the used classes

**Review by geoand — APPROVED** [review]

File: extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java [hunk 1]
```diff
@@ -36,6 +38,8 @@ public void findWebjarsAndCreateHandler(
                     new RouteBuildItem(webjarRootPath + "*",
                             recorder.getHandler(webjarRootPath, webjarNameToVersionMap),
                             false));
+        } else {
+            log.warn("No WebJars were found in the project. Requests to the /webjars/ path will always return 404 (Not Found)");
```
Comment (gsmet) [inline extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java:42]: Do we really need this warning? It looks a bit too intrusive? I might misunderstand it but won't you have the warning as soon as you enable the extension?

Comment (gastaldi) [reply extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java:42]: Intrusive? How so? The warning will only be displayed during build time if no webjar dependencies were are added to the build, which makes this extension useless.

Comment (gsmet) [reply extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java:42]: Sure but that's not how other extensions behave. You don't have a warn if you don't have any Hibernate ORM entities for instance.

Comment (gastaldi) [reply extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java:42]: I see, perhaps it should display the encountered webjars then? Or do you want to completely remove it?

**Pull request merged**